### PR TITLE
fix proxy_test.go file to avoid ChooseHostInterface

### PR DESCRIPTION
**Pull request by sub-mod**
@brendanburns @markturansky 
kindly review
should fix the mac tests failing

### CODE REVIEW

**Comment by markturansky** [comment]
I checked this out and successfully ran unit tests on OS X.

**Comment by satnam6502** [comment]
@brendandburns : is this an LGTM?

File: pkg/proxy/proxier.go [hunk 1]
```diff
@@ -325,6 +325,10 @@ func NewProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables.In
 		return nil
 	}
 	glog.Infof("Setting Proxy IP to %v", hostIP)
+	return CreateProxier(loadBalancer, listenIP, iptables, hostIP)
+}
+
+func CreateProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables.Interface, hostIP net.IP) *Proxier {
```
Comment (thockin) [inline pkg/proxy/proxier.go]: This should not be a public symbol - createProxier() or newProxierInternal() please

Comment (sub-mod) [inline pkg/proxy/proxier.go]: @thockin fixed it as part of #5497

**Pull request merged**